src/foam/lib/json/JSONParser.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
```

Comment:
2020

Reply:
This was converted from java to js - so it did have a copyright already.